foyer/forcefield.py

```diff
@@ -690,7 +714,7 @@ def apply(
         assert_angle_params=True,
         assert_dihedral_params=True,
         assert_improper_params=False,
-        combining_rule="geometric",
+        combining_rule="lorentz",
```

Comment:
I probably am just forgetting something, but why are we going back to `lorentz` as default?

Reply:
This could go either way; my thinking is that it's a more reasonable default value to have as it's more common. If I was presented a force field and forced to assume a combining rule, it's what I'd go with.

If the force field has geometric specified, that overrides the argument here. And `oplsaa.xml` has it, so it should be using geometric.

Reply:
Makes sense, so that works for me! Thanks!